spotbugs-tests/src/test/java/edu/umd/cs/findbugs/detect/Issue2749Test.java

```diff
@@ -0,0 +1,24 @@
+package edu.umd.cs.findbugs.detect;
+
+import static org.hamcrest.Matchers.emptyIterable;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnJre;
+import org.junit.jupiter.api.condition.JRE;
+
+import edu.umd.cs.findbugs.AbstractIntegrationTest;
+
+class Issue2749Test extends AbstractIntegrationTest {
+    @Test
+    @DisabledOnJre(JRE.JAVA_8)
+    void testIssue() {
+        performAnalysis(
+                "../java11/ghIssues/Issue2749$WithMethodHandles.class",
+                "../java11/ghIssues/Issue2749$WithVarHandle$Value.class",
+                "../java11/ghIssues/Issue2749$WithVarHandle.class",
+                "../java11/ghIssues/Issue2749.class");
+        assertThat(getBugCollection(), is(emptyIterable()));
```

Comment:
Instead of an empty iterable can you please specify the relevant bugcode(s)? For this there are some assert functions in AbstractIntegrationTest, e.g. [assertNoBugType()](https://github.com/spotbugs/spotbugs/blob/master/spotbugs-tests/src/test/java/edu/umd/cs/findbugs/AbstractIntegrationTest.java#L158). 
This would also help in the build failing because of the failing tests.

Reply:
Understood, I will be pushing more patches to update the implementation and tests.

Nevertheless, I want to keep this assertion in place to the extent possible. At the end of the day, I want SpotBugs to to acknowledge parts of Issue2749 as fully-compliant code -- i.e. as a downstream user I want to know that this code example is fully valid and SpotBugs won't start complaining about it in the future.

I understand that is not how the test suite works right now, but as a downstream user I need SpotBugs to start providing these sorts of guarantees for Real Code(tm).